### PR TITLE
Took 271 anchor presets

### PR DESCRIPTION
**Pull request by fcturan20**
[TOOK-271](https://oyunteknolojileri.atlassian.net/browse/TOOK-271?atlOrigin=eyJpIjoiNThjZGNlNzQ1MWFkNDI2M2E4YmQ3YWJkMjczYmE5OTMiLCJwIjoiaiJ9)

### CODE REVIEW

File: Editor/PropInspector.cpp [hunk 1]
```diff
@@ -764,7 +764,136 @@ namespace ToolKit
...
+                    uint subMode  = anchorPreset % 4;
+                    uint mainMode = (anchorPreset - subMode) / 4;
+                    // Horizontal Modes
+                    if (mainMode < 3)
+                    {
+                      // 0: Left, 1: Middle, 2: Right, 3: Side-to-Side
+                      uint anchorHorizontalPos = subMode;
+                      // 0: Top, 1: Middle, 2: Bottom
+                      uint anchorVerticalPos = mainMode;
+                      // Horizontal Size = %30 in all cases
```
Comment (rbellek) [inline Editor/PropInspector.cpp:826]: We don't set a percentage like "30%".
Stretch presets are thin as line.

File: Editor/PropInspector.cpp [hunk 2]
```diff
@@ -764,7 +764,136 @@ namespace ToolKit
...
+                      case 2:
+                        gatherAtRight();
+                        break;
+                      case 3:
+                        scatterToSide(true);
+                        break;
+                      default:
+                        assert(0 && "This shouldn't happen!");
+                      }
+                      // Vertical Size = %30 in all cases
```
Comment (rbellek) [inline Editor/PropInspector.cpp:844]: We don't set a percentage like "30%".
Stretch presets are thin as line.

File: Editor/PropInspector.cpp [hunk 3]
```diff
@@ -764,7 +764,136 @@ namespace ToolKit
...
+                   itemIndx++)
+              {
+                ImGui::TableNextColumn();
+                ImGui::PushID(itemIndx);
+                if (ImGui::ImageButton((ImTextureID) UI::m_anchorPresetIcons
+                                           .m_presetImages[itemIndx]
+                                           ->m_textureId,
+                                       Vec2(32, 32)))
+                {
+                  auto changeAnchor = [surface](uint anchorPreset) {
```
Comment (rbellek) [inline Editor/PropInspector.cpp:787]: It doesn't has to be too complex. You may just set the correct preset values into the array. Anyway, no problem with this block.

**Review by rbellek — CHANGES_REQUESTED** [review]
Preset icons must be created by us. You can use paint for a simple illustration (flat design maybe?), then we should get a better looking one in the future.

**Review by afraidofdark — CHANGES_REQUESTED** [review]

**Pull request merged**